tests/system/plugin-reachable-vulns.test.ts

```diff
@@ -7,6 +7,7 @@ import { CallGraph, CallGraphError } from '@snyk/cli-interface/legacy/common';
 import * as plugin from '../../lib';
 import { readFixtureJSON } from '../file-helper';
 import { SinglePackageResult } from '@snyk/cli-interface/legacy/plugin';
+import { CallGraphGenerationError } from '@snyk/java-call-graph-builder/dist/errors';
```

Comment:
did you need to use `dist` here? interesting

Reply:
I think that's because the error itself is not exported from the module? Not sure though.

Reply:
Should `CallGraphGenerationError` be used on the CLI?

Reply:
It's just a test. I can probably refactor it.